doc/ssl/SSL_CTX_set_tlsext_status_cb.pod

```diff
@@ -16,8 +21,10 @@ Certificate Status Request functions
  long SSL_CTX_set_tlsext_status_arg(SSL_CTX *ctx, void *arg);
 
  long SSL_CTX_set_tlsext_status_type(SSL_CTX *ctx, int type);
+ long SSL_CTX_get_tlsext_status_type(SSL_CTX *ctx);
 
  long SSL_set_tlsext_status_type(SSL *s, int type);
+ long SSL_get_tlsext_status_type(SSL *s);
```

Comment:
@mattcaswell, but "because these are macros" is a backwards argument.

1) Why are we still implementing new functions as macros at all, therefore forgoing type safety, and making reasoning about the dependencies harder for ourselves?
2) Why do the getters need to go through SSL_ctrl?
3) Why does SSL_ctrl need to support getters? (Note this is different from 2. The two could be separate if need be. Or the ctrl could be calling the getter and not the other way around.)
4) Why are we overloading the same ctrl return value for success/fail and a proper integral return values?
5) Why do we need SSL_ctrl at all?

I can see that it's too late to bring SSL_ctrl to purgatory for 1.1.0 but perhaps we can come up with a better answer and plan than "because it's always been like this"? If the ctrl needs to exist (why??) then so be it, but bending new public APIs to its old buggy internals makes no sense to me.